### DISCUSSION TITLE
Clarification on the Move to Closed-Source

### DISCUSSION THREAD

**Discussion by jacob-willden** [FrostCo/AdvancedProfanityFilter#497] [Q&A] [answered]
Hello Mr. Frost. Thank you so much for creating and maintaining this fantastic browser extension! I have a question about the recent move to version 3.0, where you decided to move the audio-filtering portion of the source code out of the open-source repository to prevent further GPL violations. You mentioned in the [3.0 release notes](https://github.com/FrostCo/AdvancedProfanityFilter/releases/tag/3.0.0) that the versions of the extension from the "official store" will keep the audio muting feature. Does the "official store" include the Chrome, Firefox, and Safari extension stores? Thanks in advance!

**Comment by richardfrost** [comment, marked as answer]
Yes @jacob-willden, it does include the stores you listed as well as the Microsoft Edge and Opera storefronts as well.

**Comment by jacob-willden** [comment]
Cool! Thanks for the clarification!
